Tasks/AzurePowerShellV4/AzurePowerShell.ps1

```diff
@@ -23,33 +21,12 @@ if ($scriptArguments -match '[\r\n]') {
     throw (Get-VstsLocString -Key InvalidScriptArguments0 -ArgumentList $scriptArguments)
 }
 
-# string constants
-$otherVersion = "OtherVersion"
-$latestVersion = "LatestVersion"
-
-if ($targetAzurePs -eq $otherVersion) {
-    if ($customTargetAzurePs -eq $null) {
-        throw (Get-VstsLocString -Key InvalidAzurePsVersion $customTargetAzurePs)
-    } else {
-        $targetAzurePs = $customTargetAzurePs.Trim()        
-    }
-}
-
-$pattern = "^[0-9]+\.[0-9]+\.[0-9]+$"
-$regex = New-Object -TypeName System.Text.RegularExpressions.Regex -ArgumentList $pattern
-
-if ($targetAzurePs -eq $latestVersion) {
-    $targetAzurePs = ""
-} elseif (-not($regex.IsMatch($targetAzurePs))) {
-    throw (Get-VstsLocString -Key InvalidAzurePsVersion -ArgumentList $targetAzurePs)
-}
-
-. "$PSScriptRoot\Utility.ps1"
-
 $serviceName = Get-VstsInput -Name ConnectedServiceNameARM -Require
 $endpoint = Get-VstsEndpoint -Name $serviceName -Require
 
-Update-PSModulePathForHostedAgent -targetAzurePs $targetAzurePs
```

Comment:
Where are we updating the PSModule path with targetAzurePS?

Reply:
done